daemon/src/db.rs

```diff
@@ -315,6 +317,28 @@ impl Connection {
                 yield self.load_closed_cfd(id, args.clone()).await
                     .with_context(|| format!("Failed to load closed CFD {id}"));
             }
+
+            let mut conn = self.inner.acquire().await?;
+
+            let ids = sqlx::query!(
+                r#"
+                SELECT
+                    uuid as "uuid: model::OrderId"
+                FROM
+                    failed_cfds
+                "#
+            )
+            .fetch_all(&mut *conn)
```

Comment:
I recently learned that there is also `fetch` which returns a `Stream`! That would make the whole thing even more "iteratively"!

Reply:
Thanks for the tip!

How do you think it would interact with the fact that we would be using the same `Connection` to generate this stream and to load CFDs?

Reply:
Yeah you would have to use the same connection likely. One thing that you might be able to do is do the mapping right on the stream that is being returned instead of collecting it all first. Doesn't make much of a difference though I think!

Reply:
In the interest of getting this merged before I encounter more conflicts I vote for merging this as is and I can look at this suggestion in a follow-up PR.